test/stripe/subscriptions/invoice_test.exs

```diff
@@ -15,48 +15,52 @@ defmodule Stripe.InvoiceTest do
     end
   end
 
-  describe "upcoming/1" do
-    test "retrieves an upcoming invoice for a customer" do
+  describe "create_preview/1" do
+    test "creates a preview invoice for a customer" do
       params = %{customer: "cus_123", subscription: "sub_123"}
-      assert {:ok, %Stripe.Invoice{}} = Stripe.Invoice.upcoming(params)
```

Comment:
what happened to `upcoming`?

Reply:
I was replaced by with the `Create Preview Invoice API`
https://docs.stripe.com/changelog/basil/2025-03-31/invoice-preview-api-deprecations#what%E2%80%99s-new